cassandane/Cassandane/Cyrus/SearchSquat.pm

```diff
@@ -438,4 +438,28 @@ sub test_relocate_legacy_nosearchdb
     $self->assert_equals(0, scalar @files);
 }
 
+sub test_unindexed
+    :SearchEngineSquat :min_version_3_4
+{
+    my ($self) = @_;
+    my $imap = $self->{store}->get_client();
+
+    $self->make_message("needle 1", body => "needle") || die;
+    $self->make_message("xxxxxx 2", body => "xxxxxx") || die;
+
+    $self->{instance}->run_command({cyrus => 1}, 'squatter');
```

Comment:
You could use `$self->run_squatter();` here now ([implementation](https://github.com/cyrusimap/cyrus-imapd/blob/master/cassandane/Cassandane/Cyrus/SearchSquat.pm#L72-L91)).  I added it just recently for the other squat fixes.

Reply:
I saw, just didn't bother. But I updated the code now to use it.